.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,14 +1,11 @@
-### Please check if the PR fulfills these requirements:
-
-* [ ] Tests for the changes have been added/updated (if possible)
-* [ ] Documentation has been updated/added if relevant
```

Comment:
I would think documentation edits should still be mentioned somewhere here. Especially for new devs and it's always a good reminder for experienced devs